test/cqlpy/test_describe.py

```diff
@@ -183,19 +183,31 @@ def test_desc_scylla_keyspace(scylla_only, cql, random_seed):
 
 # Test that `DESC TABLE {tbl}` contains appropriate create statement for table
 # This test compares the content of `system_schema.tables` and `system_schema.columns` tables.
-def test_desc_table(cql, test_keyspace, random_seed, has_tablets):
+@pytest.mark.parametrize("with_tablet_hints", [False, True])
+def test_desc_table(cql, test_keyspace, random_seed, has_tablets, with_tablet_hints:bool):
     if has_tablets:  # issue #18180
         global counter_table_chance
         counter_table_chance = 0
-    with new_random_table(cql, test_keyspace) as tbl:
+    tablet_hints = {}
+    if with_tablet_hints:
+        tablet_hints = {
+            'min_tablet_count': '100',
+            'min_per_shard_tablet_count': '10.5',
+            'expected_data_size_in_gb': '50',
+        }
+    with new_random_table(cql, test_keyspace, tablet_hints=tablet_hints) as tbl:
         desc = cql.execute(f"DESC TABLE {tbl}")
         desc_stmt = desc.one().create_statement
 
         new_tbl = f"{test_keyspace}.{unique_name()}"
         new_desc_stmt = desc_stmt.replace(tbl, new_tbl)
 
         try:
-            cql.execute(new_desc_stmt)
+            try:
+                cql.execute(new_desc_stmt)
+            except Exception as e:
+                print(f"{new_desc_stmt=}: {e}")
+                raise
```

Comment:
Please reconsider this change. pytest already shows stack traces beautifully, you shouldn't need to do things like this...

Reply:
I did this since this default stack trace was confusing, but I can revert his hunk.
It was needed for debugging

Reply:
It was "confusing" because it was missing information, or because you're just not used to it?

If it's the latter, we don't need this change. Future developers who look at this test will not necessarily need to debug failures in this specific spot, and wherever the failure happens to be, hopefully pytest's stack trace will be good enough.

Reply:
Dropped in v3